pkg/types/clusterconfig/cluster_config.go

```diff
@@ -935,19 +945,41 @@ func (ng *NodeGroup) validateNodeGroup(awsClient *aws.Client, region string) err
 		return errors.Wrap(ErrorInstanceTypeNotSupportedInRegion(primaryInstanceType, region), InstanceTypeKey)
 	}
 
-	// Throw error if IOPS defined for other storage than io1
-	if ng.InstanceVolumeType != IO1VolumeType && ng.InstanceVolumeIOPS != nil {
+	// throw error if IOPS defined for other storage than io1/gp3
+	if ng.InstanceVolumeType != IO1VolumeType && ng.InstanceVolumeType != GP3VolumeType && ng.InstanceVolumeIOPS != nil {
 		return ErrorIOPSNotSupported(ng.InstanceVolumeType)
 	}
 
-	if ng.InstanceVolumeType == IO1VolumeType && ng.InstanceVolumeIOPS != nil {
-		if *ng.InstanceVolumeIOPS > ng.InstanceVolumeSize*50 {
-			return ErrorIOPSTooLarge(*ng.InstanceVolumeIOPS, ng.InstanceVolumeSize)
+	// throw error if throughput defined for other storage than gp3
+	if ng.InstanceVolumeType != GP3VolumeType && ng.InstanceVolumeThroughput != nil {
+		return ErrorThroughputNotSupported(ng.InstanceVolumeType)
+	}
+
+	if ng.InstanceVolumeIOPS != nil {
+		if ng.InstanceVolumeType == IO1VolumeType {
+			if *ng.InstanceVolumeIOPS > _highestIOPSForIO1VolumeType {
+				return ErrorIOPSTooLarge(*ng.InstanceVolumeIOPS, _highestIOPSForIO1VolumeType)
+			}
+			if *ng.InstanceVolumeIOPS > ng.InstanceVolumeSize*50 {
+				return ErrorIOPSTooLargeDueToVolumeSize(*ng.InstanceVolumeIOPS, ng.InstanceVolumeSize)
+			}
+		} else {
+			if *ng.InstanceVolumeIOPS > _highestIOPSForGP3VolumeType {
```

Comment:
Add validation: Maximum ratio of 0.25:1 is permitted between Throughput (MiBps) and IOPS.